pylasu/parsing/results.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import List
+
+from antlr4 import ParserRuleContext, Token
+
+from pylasu.model import Source
+from pylasu.validation.validation import WithIssues, IssueType, Issue
+
+
+@dataclass
+class FirstStageResult(WithIssues):
+    parse_tree: ParserRuleContext
+
+
+@dataclass
+class LexingResult(WithIssues):
```

Comment:
Same as `FirstStageResult` and every other `WithIssues` subclass

Reply:
Note that the `issues` list here would not be required if following the comment concerning `WithIssues` below